src/manifolds/MetricManifold.jl

```diff
@@ -51,6 +51,126 @@ inner product ``g(X, X) > 0`` whenever ``X`` is not the zero vector.
 """
 abstract type RiemannianMetric <: AbstractMetric end
 
+@doc raw"""
+    change_gradient(M::AbstractManifold, G2::AbstractMetric, p, X)
+
+Convert the gradint `X` at `p` on the[`AbstractManifold`](@ref) `M` from one metric to another.
+
+Assume that for a real-valued function ``f: \mathcal M \to ℝ`` we are given the Riesz representer of the differential with respect to the metric ``g_2`` i.e.
```

Comment:
I think gradient and Riesz representer of the differential are the same thing? Maybe it would be nice to mention it here for clarity?

Reply:
Sure. The gradient is the Riesz representer of the Differential. I will mention that and maybe even link Riesz to the wikipedia article – good idea.